lib/Pipfile

```diff
@@ -41,8 +41,7 @@ attrs = "*"
 base58 = "*"
 blinker = "*"
 cachetools = ">=4.0"
-# Click 8.0 has a new incompatible API
-click = ">=7.0, <8.0"
+click = "*"
```

Comment:
Do we know why we required click 7+? We still want the lower bound if the reason for it (such as some api we use introduced then) remains.

Reply:
Ah yea...I was excited to unpin it, I didn't realize we had a min version. Thanks for the check.